overlord/snapstate/snapstate_test.go

```diff
@@ -119,6 +121,24 @@ func (s *snapmgrBaseTest) logTasks(c *C) {
 
 var fakeRevDateEpoch = time.Date(2018, 1, 0, 0, 0, 0, 0, time.UTC)
 
+func (s *snapmgrBaseTest) mockSystemctlCallsUpdateMounts(c *C) (restore func()) {
+	r := systemd.MockSystemctl(func(args ...string) ([]byte, error) {
+		if len(args) == 1 && args[0] == "daemon-reload" {
+			return []byte(""), nil
+		}
+		if len(args) == 3 && args[0] == "--no-reload" && args[1] == "enable" {
+			return []byte(""), nil
+		}
+		if len(args) == 2 && args[0] == "reload-or-restart" {
+			return []byte(""), nil
```

Comment:
should we also track/count if reload-or-restart is called to distinguish situations in  the new tests below?